packages/Manager/src/main/kotlin/nz/govt/eop/tasks/GeoJsonManifestUpdater.kt

```diff
@@ -13,9 +14,10 @@ class GeoJsonManifestUpdater(val context: DSLContext, val manifest: GeoJsonQuery
   private val logger = KotlinLogging.logger {}
 
   @Scheduled(fixedDelay = 1, timeUnit = TimeUnit.MINUTES)
+  @SchedulerLock(name = "geoJsonManifestUpdater")
```

Comment:
@smozely does this prevent more than one running concurrently?

Reply:
@vimto yeah, it ends up using the shedlock table in Postgres to lock against. And has some magic to make sure things don't get locked forever.